src/control/cmd/daos_agent/procmon.go

```diff
@@ -76,7 +76,7 @@ const MonWaitTime = 3 * time.Second
 // monitorProcess is used by procMon to kick off monitoring individual processes
 // under their own child context to allow for terminating individual monitoring routines.
 func (p *procInfo) monitorProcess(ctx context.Context) {
-	Ino, err := getProcPidInode(p.pid)
+	_, err := getProcPidInode(p.pid)
```

Comment:
This seems a bit weird -- why not just remove the call altogether? If we're not using the information returned by the function call, why would we handle an error originating in the function?

Reply:
They are using the detection of /proc/pid still being there just not the inode number from it. So they are using it to detect that /proc/pid still exists not necessarially that it represents the same process.

Reply:
OK, fair enough. If and until this code is rewritten, though, this is going to be a bit confusing. It would be nice to just check for the pid file directly rather than using this mostly-dead function. At a minimum, there should at least be a comment explaining what's going on here.

Reply:
There doesn't seem to be any reason for getting the inode of a pid directory, so there seems to be no benefit keeping this function in place.  If we don't remove it now we'll just have to remove it later during cleanup.

Reply:
Agreed. Returning the inum is at least not necessary and can be removed from the function.

Reply:
Modified the function just to check that stat doesn't return an error and ignores the actual stat struct returned.